src/blog/better-class-features.md

```diff
@@ -0,0 +1,332 @@
+---
+title: 'Better support for class features'
```

Comment:
```suggestion
title: 'Faster class field initialization'
```

I think the original title made more sense with the snapshot stuff also in scope, which we've since moved out to the Node article.

Please also rename the blog markdown file + asset folder name.

Reply:
Since the post also includes how initialization of private brands was optimized maybe “Faster instance initialization with new class features”  (and “faster-class-features” for the file name) would be better?

Reply:
I took the liberty to rename the title to `Faster initialization of instances with new class features` and added a link to the Node.js release announcement before I log off - please let me know if you think that the original suggestion is better, thanks

Reply:
wfm, thanks